doc/release/1.4.0-notes.rst

```diff
@@ -392,6 +391,7 @@ Authors
 =======
 
 * @endolith
+* @wenhui-prudencemed +
```

Comment:
@wenhui-prudencemed I've added your GitHub handle to the updated release notes---apologies in advance if you have another preference for naming & we end up merging first (fairly likely considering recent delays!).